sql/expression/function/time_test.go

```diff
@@ -243,7 +243,7 @@ func TestTime_DayOfWeek(t *testing.T) {
 		{"null date", sql.NewRow(nil), nil, false},
 		{"invalid type", sql.NewRow([]byte{0, 1, 2}), nil, false},
 		{"date as string", sql.NewRow(stringDate), int32(3), false},
-		{"date as time", sql.NewRow(time.Now()), int32(time.Now().UTC().Weekday()+1) % 7, false},
+		{"date as time", sql.NewRow(time.Now()), int32(time.Now().UTC().Weekday() + 1), false},
```

Comment:
if weekday == 6 it would return 7, MySQL weekday index range is [0, 6] https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_weekday

Reply:
I changed it because tests were failing

Reply:
I changed it because tests were failing

Reply:
This test is testing  a DayOfWeek (not weekday): https://dev.mysql.com/doc/refman/8.0/en/date-and-time-functions.html#function_dayofweek

just expected value was specified by go's weekday, so there is no need for `%`